Volume/Supporting Views/VolumeMessage.swift

```diff
@@ -11,11 +11,15 @@ import SwiftUI
 enum Message {
     case noBookmarks
     case upToDate
+    case noFollowingHome
+    case noFollowingPublications
```

Comment:
This is a different convention than the rest of the project.

Reply:
Since the noFollowing message is different based on if its on the home page or on the publications page, I need two different cases for this.